waves-integration/src/main/scala/com/wavesplatform/dex/domain/order/OrderV3.scala

```diff
@@ -41,21 +41,43 @@ case class OrderV3(@ApiModelProperty(
                    matcherFee: Long,
                    @ApiModelProperty(
                      name = "matcherFeeAssetId",
-                     value = "Base58 encoded Matcher fee asset ID, equals to WAVES for Order versions 1 and 2",
+                     value = "Base58 encoded Matcher fee asset ID. Waves is used if field isn't specified",
                      dataType = "string",
                      example = "8LQW8f7P5d5PZM7GtZEBgaqRPGSzS3DfPuiXrURJ4AJS",
-                     required = true
+                     required = false
                    )
                    override val feeAsset: Asset,
                    @ApiModelProperty(
-                     value = "Order proofs as Base58 encoded signatures list",
+                     value =
+                       "Order proofs as Base58 encoded signatures list. " +
+                         "If Sender's account doesn't have script, put Order's signature as the first proof.\n" +
+                         "Signature = Base58 encoded Curve25519.sign(senderPrivateKey, concat(\n" +
+                         "- version,\n" +
+                         "- bytesOf(senderPublicKey),\n" +
+                         "- bytesOf(matcherPublicKey),\n" +
+                         "- bytesOf(amountAsset),\n" +
+                         "- bytesOf(priceAsset),\n" +
+                         "- bytesOf(orderType), // = 0 for buy, 1 for sell\n" +
+                         "- bigEndianBytes(price),\n" +
+                         "- bigEndianBytes(amount),\n" +
+                         "- bigEndianBytes(timestamp),\n" +
+                         "- bigEndianBytes(expiration),\n" +
+                         "- bigEndianBytes(matcherFee),\n" +
+                         "- bytesOf(matcherFeeAssetId)\n" +
+                         "))",
```

Comment:
Focus-pocus:
```scala
val message =
  """foo
bar
 baz
  qux"""
```
all spaces and newlines are preserved!

Reply:
Doesnt't work for swagger, actually

Reply:
Fixed